central/development/service/service_impl.go

```diff
@@ -29,6 +31,18 @@ var (
 	x509Err = x509.UnknownAuthorityError{}.Error()
 )
 
+var (
+	authorizer = perrpc.FromMap(map[authz.Authorizer][]string{
+		user.Authenticated(): {
+			"/central.DevelopmentService/ReplicateImage",
+			"/central.DevelopmentService/URLHasValidCert",
+			"/central.DevelopmentService/RandomData",
+			"/central.DevelopmentService/EnvVars",
+			"/central.DevelopmentService/ReconciliationStatsByCluster",
```

Comment:
These are for dev builds only. Not sure we want to introduce authn. Are you concerned that we expose sensitive info on stagingdb?

Reply:
I think having some sort of security on dev builds wouldn't hurt - why let anyone mess around with publicly available instances(stagingdb is a good example) if we can close this gap without hurting dev experience?

Reply:
These services are tools for developers.
I think `ReplicateImage` requires `Image READ` to do anything meaningful.
The `EnvVars` service could expose sensitive information.
Although these services should not be exposed in customer environments, I think it does not hurt to restrict access.

Reply:
I'd rather protect the security posture of potential dev image deployment.
I'll make the internal development endpoints require full admin role, and adjust the e2e test to target these as admin user.